ADOPTERS.md

```diff
@@ -8,6 +8,7 @@ URLs if possible) and [submit a pull request][1]!
 
 * [Attendify](https://attendify.com)
 * [Box](https://www.box.com/)
+* [Buoyant](http://buoyant.io/)
```

Comment:
Buøyant is actually our preference (in honor of Finagle's Norse heritage). But we're fine with whatever.